cookbooks/clearwater/templates/default/shared_config.erb

```diff
@@ -10,6 +10,22 @@ sas_server=<%= @node[:clearwater][:sas_server] or "0.0.0.0" %>
 enum_server=<%= @node[:clearwater][:enum_server] or "" %>
 alias_list=<%= @alias_list %>
 
+<% if not @database.nil? %>
```

Comment:
I was following the format as later in the file, e.g. 
```
# HSS configuration
 <% if not @hss.nil? %>hss_hostname=<%= @hss %>...
```
I wasn't sure if there was some idiosyncratic reason one couldn't just check `if @database`; does just checking `if @database` see if there is a `database` variable, as opposed to whether it has a real value assigned?